QA/README.md

```diff
@@ -0,0 +1,79 @@
+# CherryTwist - Quality Assurance
+
+This document provides overview of "Cherrytwist" quality assurance and testing activities. Some of them are in progress and some are to be started.
+
+# Process
+ 
+- Perform functional/non-functional testing types on reviewed "github" issues
+    - Depending on the "github" issue scope, testing is performed on different environments 
+- Log defects, communicate  and retest them
+- Perform regression testing as part of "github" issues testing
+- Automate/maintain test cases on different levels
+- Test data management
+
+
+# Testing types and levels
+
+"Cherrytwist" consists of multiple services. The following test types are being applied among the different test levels:
+
+## "Server" - testing types and levels:
+- ### Functional testing: 
+    - Unit testing<sup>*</sup>
+    - [GraphQL API integration tests](https://github.com/cherrytwist/Server/tree/develop/test)
+    - GraphQL Exploratory testing
+        
+- ### Non-functional testing:
+    - [Performance testing](https://github.com/cherrytwist/Server/tree/develop/test/performance)
+    - Security testing<sup>*</sup>
+    - Database migration<sup>*</sup>
+
+- ### Static testing:
+    - Code review
+    - Static code analyses tools<sup>*</sup>
+ 
+## "Client.Web" - testing types and levels:
```

Comment:
Please put in a comment that the focus for QA is on the server - client QA to be ramped up